single-solution/SolutionPage.js

```diff
@@ -1,158 +1,282 @@
+/**
+ * A constant to use in this form. This number represents the accepted percentage of error
+ * when calculating mass or volume to add.
+ * @type {number} Allowed percent error to consider correct. (0-100)
+ */
 var ACCEPTED_PERCENT_ERROR = 1;
 
+$(function(){
+
+    hideAlert();
+
+});
+
 function next_check(page){
     try {
 
```

Comment:
Overhauled next_check(page) to include more error checking and greater feedback fidelity.